src/test/java/edu/kis/powp/jobs2d/events/SelectVisitorTestListener.java

```diff
@@ -0,0 +1,78 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.CommandUsageCounterVisitor;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class SelectVisitorTestListener implements ActionListener {
```

Comment:
Name should explain the function: e.g., SelectCommandUsageCounterVisitorTestListener.